README.adoc

```diff
@@ -197,3 +198,16 @@ oe-selftest --run-tests updater
 ```
 
 For more information about oe-selftest, including details about how to run individual test modules or classes, please refer to the https://wiki.yoctoproject.org/wiki/Oe-selftest[Yocto Project wiki].
+
+== Manual provisoning
+As described in <<sota-related-variables-in-localconf,SOTA-related variables in local.conf>> section you can set `SOTA_DEPLOY_CREDENTIALS` to `0` to prevent deploying credentials to the built `wic` image. In this case you get a generic image that you can use e.g. on a production line to flash a series of devices. The cost of this approach is that this image is half-backed and should be provisioned before it can connect to the backend.
+
+Provisioning procedure depends on your provisioning recipe, i.e. the value of `SOTA_CLIENT_PROV` (equal to `aktualizr-auto-prov` by default).
+* For `aktualizr-auto-prov` put your `credentials.zip` to `/ostree/deploy/poky/var/sota/sota_provisioning_credentials.zip`
+* For `aktualizr-ca-implicit-prov`
+** put URL to the used backend (together with protocol prefix and port number) to `/var/sota/gateway.url`. If you're using HERE OTA Connect, you can find the URL in `autoprov.url` file in your credentials archive.
+** put client certificate, private key and root CA certificate (for the *server*, not for the *device*) to `/var/sota/import_client.pem`, `/var/sota/import_pkey.pem` and `/var/sota/import_root.crt` respectively.
+* For  `aktualizr-hsm-prov`
+** put URL to the used backend (together with protocol prefix and port number) to `/var/sota/gateway.url`. If you're using HERE OTA Connect, you can find the URL in `autoprov.url` file in your credentials archive.
+** put root CA certificate (for the *server*, not for the *device*) to `/var/sota/import_root.crt`.
+** put client certificate and private key to slots 1 and 2 of the PKCS#11-compatible device.
```

Comment:
Nice write-up. Is the `/ostree/deploy/poky` prefix really necessary for the autoprov bit? And shouldn't the import paths be `/var/sota/import/xyx` instead of `/var/sota/import_xyz`?

Reply:
> Is the `/ostree/deploy/poky` prefix really necessary for the autoprov bit?

Good point. I was describing how it is done with sd card mounted on the build machine, and there it is necessary. I should make it clear probably and also describe how it can be done with `scp` and friends.

> And shouldn't the import paths be /var/sota/import/xyx instead of /var/sota/import_xyz

Thanks, I forgot to update this part.